lib/mayaUsd/ufe/StagesSubject.cpp

```diff
@@ -307,14 +307,6 @@ void StagesSubject::afterOpen()
             }
         });
     fStageListeners.clear();
-
-    // Set up our stage to proxy shape UFE path (and reverse)
-    // mapping.  We do this with the following steps:
-    // - get all proxyShape nodes in the scene.
-    // - get their Dag paths.
-    // - convert the Dag paths to UFE paths.
-    // - get their stage.
-    g_StageMap.setDirty();
```

Comment:
What happened with this map. I don't see anywhere that is kept up-to-date now, but it is still being referenced in Utils.cpp.

Reply:
The map now gets updated automatically when it detects an attempt to access out-of--date data. But you are right I could remove the extern declaration at the top of the file. Also, I did not expect a review, I still need to write a unit test. :) But hey, you are very pro-active :)

Reply:
Oh, and Utils.cpp just provides function to access the content of the map, so that is normal that it still refers to it.

Reply:
Sorry never mind my comment. I didn't release at first this variable is our UsdStageMap class. I thought it was a Pixar type that we needed to add/remove items from.